spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
     describe '.new' do
-      subject(:person) { BlockScore::Person.new(attributes_for(:person)) }
+      let(:attrs)      { attributes_for(:person) }
+      subject(:person) { BlockScore::Person.new(attrs) }
 
       it { is_expected.not_to be_persisted }
       its(:class) { should be BlockScore::Person }
+      its(:question_sets) { should be_an_instance_of BlockScore::Collection }
```

Comment:
I might suggest a separate `describe '#question_sets'` block with this. I generally would have the `.new` describe block focus on describing the behaviour and side effects of the specified method. The fact that there's a `#question_sets` method available on the object isn't really pertinent to the constructor.